Application/Business/UserServices/UserServices.cs

```diff
@@ -25,39 +30,31 @@ public async Task<int> Count()
     }
     public async Task Create(UserEntity user) {
         var userList = List();
```

Comment:
should use the function _repo.firstOrDefaultAsync to check either user in db or not. Let the db handle this for you instead of list all users from db and traverse to find the duplicated